packages/sprotty/src/base/commands/command.ts

```diff
@@ -52,6 +52,17 @@ export interface ICommand {
     redo(context: CommandExecutionContext): CommandReturn
 }
 
+/**
+ * A stoppable commands execution (e.g. one that starts an animation) can be interrupted.
+ */
+export interface IStoppableCommand extends ICommand {
+    stopExecution(): void
+}
+
+export function isStoppableCommand(command: any): command is IStoppableCommand {
+    return command && 'stopExecution' in command && typeof command.stopExecution === 'function';
```

Comment:
You can use the utility function `hasOwnProperty` from sprotty-protocol here.

Reply:
That does not work since the stopExecution is a member of the command objects prototype (MergeableCommand) and not a direct member at runtime.